### PR TITLE
Add weekly meeting minutes, 2018-0-13

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Minutes from 13 September 2018.

There were a few points I may have failed to capture:

- The organiser/URL for the ANL exascale workshop (@sextonkennedy)
- Some points about what labs could or could not cover in registration costs for the workshop (from Daniel)

(However, these could be fixed later, probably should not block on them.)

### CODE REVIEW

File: organization/_posts/2018/2018-09-13-coordination.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+Activity updates
+================
+
+Training
+--------
+-   [Formats discussion](https://groups.google.com/forum/#!topic/hsf-tech-forum/k9V-0buFD0Q)
+    (tech list) was very useful.
+    -   RISE extension for Juypter notebooks seems ideal for live
+        presentations.
+-   Dedicate one of our next meeting to training activities?
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-09-13-coordination.md]: meetings

File: organization/_posts/2018/2018-09-13-coordination.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        -   How much ROOT progress in the I/O layer?
+        -   When packaging WG would reach conclusions?
+        -   Are other FAs in supporting software (in the wake of the
+            IRIS-HEP news)?
+            -   New WGs are important in defining concrete work plans.
+            -   LHCC has been asked to offer general support for work
+                plans, that helps this to happen.
+        -   LHCC have not yet set a date for the review of HL-LHC (may
+            not be before next summer).
+-   COMCHA Presentation:
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-09-13-coordination.md]: Maybe add that COMCHA is a Spanish activity around COMputing CHAllenges?

File: organization/_posts/2018/2018-09-13-coordination.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+---------
+-   [Next meeting](https://indico.cern.ch/event/754985/),
+    September 19.
+    -   Report from Hadrien Grasland about his experience using Spack to
+        build ACTS.
+
+Frameworks
+----------
+-   Workshop at ANL arranged for discuss I/O for exascale. Recommended
+    re-investigating OpenMP. HPC community have invested here and now
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-09-13-coordination.md]: setence "HPC ..." does not read too well.

**Comment by graeme-a-stewart** [comment]
Thanks for the good points. Should be ok now. (I see the RSE broke their SSL setup, for now I think we just leave that.)

**Pull request merged**